stor/utils.py

```diff
@@ -260,17 +260,28 @@ def find_dx_class(p):
     from stor.dx import DXPath, DXCanonicalPath, DXVirtualPath
     parts = p[len(DXPath.drive):].split('/')
     if not parts or not parts[0]:
-        assert False, 'Only project level paths are supported for DX currently'
+        raise ValueError('Must specify a project for DXPath')
+
+    # first part can be 'proj:file' or 'proj:' or 'proj'
+    parts_first = parts[0].split(':')
+    parts[0] = parts_first[0]
+    if len(parts_first) > 1 and parts_first[1]:
+        parts.insert(1, parts_first[1])
+
     try:
         # verify_string_dxid returns None if success, raises error if failed
-        if not verify_string_dxid(parts[0][:-1], 'project'):
-            if len(parts) > 1 and parts[1] and not verify_string_dxid(parts[1], 'file'):
+        if not verify_string_dxid(parts[0], 'project'):
```

Comment:
let's write a custom verify wrapper that looks like:

```
def _verify_dxid(.., class):
    try:
     exc
```

and invert response or call it `is_valid_dxid`, `True` if valid, `False/None` otherwise